typeDefs.js

```diff
@@ -288,14 +288,14 @@ const typeDefs = gql`
     updateUserByToken(user: UserInput!): User
 
     "Products Mutation"
-    addProducts(product: productsInput!): Products
-    deleteProducts(id: ID!): Products
-    updateProductById(id: ID!, product: productsInput!): Products
+    addProduct(product: productsInput!): Products
+    deleteProduct(id: ID!): Products
+    updateProduct(id: ID!, product: productsInput!): Products
 
     "Comments Mutation"
-    addComments(comment: commentsInput!): Comments
-    deleteCommentsById(id: ID!): Products
-    updateCommentsById(id: ID!, product: commentsInput!): Comments
+    addComment(comment: commentsInput!): Comments
+    deleteComment(id: ID!): Products
+    updateComment(id: ID!, product: commentsInput!): Comments
```

Comment:
Wrong name of the variable. It should be `comment`

Reply:
Renamed